lte/gateway/c/oai/tasks/nas/esm/sap/esm_recv.c

```diff
@@ -55,6 +51,10 @@ extern int send_modify_bearer_req(mme_ue_s1ap_id_t ue_id, ebi_t ebi);
 /*******************  L O C A L    D E F I N I T I O N S  *******************/
 /****************************************************************************/
 
+#define ERAB_SETUP_RSP_COUNTER_MAX 5
```

Comment:
Can we move these macros to esm_ebr.h and reuse across both default and dedicated bearers?